packages/keychain/src/hooks/connection.tsx

```diff
@@ -63,14 +65,22 @@ export function ConnectionProvider({ children }: PropsWithChildren) {
   }, [context, parent]);
 
   useEffect(() => {
-    if (!isIframe()) {
+    if (isIframe()) {
```

Comment:
URL search params only exists on iframe right?

Reply:
Ah this broke `slot auth login` flow. It looks like we're passing the prefund data over URL so it needs this. Just going to remove this condition check. Policies, origins, etc can just be overridden connect.